Sources/Vaux/Builders.swift

```diff
@@ -108,6 +108,18 @@ public func linkStylesheet(url: String) -> HTML {
   return HTMLNode(tag: "link", child: nil).attr("href", url).attr("rel", "stylesheet")
 }
 
+/// Inserts a `<script>...</script>` element into the HTML document for inline scripting.
+/// - Parameter child: `HTML`
+public func script(code: String) -> HTML {
+    return HTMLNode(tag: "script", child: code)
+}
+
+/// Inserts a `<script src="src"></scrip>` element into the HTML document.
+/// - Parameter src: The localtion of the script to insert.
+public func linkScript(src: String) -> HTML {
```

Comment:
I don't like the use of `src` as a variable here. I would rather use something like `filepath`, which is more consistent with this use case in Vaux:

```swift
public enum VauxOutput {
  case stdout
  case file(name: String, path: String)
}
```

Additionally, I think we could re-use the `script` function, and make this a great case for parameter overloading if we decide to use a built-in type. That way, you have access to this like so:

```swift
script(code: String)
script(filepath: Filepath)
```

I think `Filepath` deserves its own definition so that I can reuse it in the example above. Let me try this out and come back to you before merging this in.

Reply:
Interesting indeed!

Reply:
Ok, if you merge in the latest from `master`, you should have access to `Filepath` now like so:

```swift
public struct Filepath {
  var name: String
  var path: String
}
```

Reply:
I will have a look!